markdown_stoplight/events.md

```diff
@@ -77,53 +78,19 @@ price: {taxType: "NET", formatted: "£40,00", amount: 40, currency: "GBP"}
 productDataSheet: null
 productId: "5954B706-E701-F357-A52D-D5809AB3F606"
 productVariationSelection: null
-productVariationValues: ""
+productVariationValues: null
 sku: "1007"
 slideshow: [{…}, {…}, {…}, {…}]
 slug: "homemade-cherry-jam"
-stockLevelClass: "in"
+stockLevelClass: "little"
```

Comment:
This one is either `in`, `little` or `out` depending on the stocklevel state of the product.

Reply:
But again is only relevant for our theming, apps should not use it but instead use `stockLevelMicrodata` or the different booleans `onStock`, `warnStock` and `outOfStock`.